prometheus/Readme.md

```diff
@@ -0,0 +1,62 @@
+# Prometheus Configuration
+
+## Default `query-max-concurrency` Setting: **4**
+
+The `query-max-concurrency` setting in Prometheus controls the maximum number of queries that can be executed concurrently.
+Setting this value appropriately is crucial to ensure that Prometheus can handle the load without overwhelming the system’s resources (CPU, memory, and disk).
+
+### Why Set `query-max-concurrency` to **4**?
+
+A concurrency setting of **4** is a reasonable starting point for most environments with moderate resource availability. Here's why:
```

Comment:
Let's describe "moderate resource availability" a bit. Suggest that as a rule of thumb we use the number of CPUs. There isn't any reason to go higher unless the workload is IO-bound.